src/effects/backends/builtin/echoeffect.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <QMap>
+#include <span>
 
```

Comment:
move this to the `.cpp`. You don't need it in the header.

Reply:
done :--)